### PR TITLE
NMS-16486: Extracted caches from DocumentEnricherImpl

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/NMS-16486

### CODE REVIEW

File: features/telemetry/protocols/cache/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+           xsi:schemaLocation="
+		http://www.osgi.org/xmlns/blueprint/v1.0.0
+		http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+		http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0
+		http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
+">
+    <!-- Configuration properties -->
+    <cm:property-placeholder id="cacheConfigProperties" persistent-id="org.opennms.features.telemetry.protocols.common" update-strategy="reload">
```
Comment (indigo423) [inline features/telemetry/protocols/cache/src/main/resources/OSGI-INF/blueprint/blueprint.xml:12]: I think it makes sense to create a documentation enhancement to add all these new configuration properties to the docs and make it a requirement when we decide to merge this into an official release. It's not a blocker for the merge into NMS-16375.

**Review by indigo423 — APPROVED** [review]
Beside the comment, nothing I can complain about :)

**Review by fooker — CHANGES_REQUESTED** [review]
Main itch is the naming. The `NodeMetadataCache` gives out `NodeInfo`. Also the package naming does not reflect the maven project structure. `NodeInfo` is located in `.../protocols/common/cache` where as `protocols/common` and `protocols/cache` both exist.

**Comment by christianpape** [comment]
> Main itch is the naming. The `NodeMetadataCache` gives out `NodeInfo`. Also the package naming does not reflect the maven project structure. `NodeInfo` is located in `.../protocols/common/cache` where as `protocols/common` and `protocols/cache` both exist.

Yes, you are totally right. The common.cache is a leftover of changes I reverted. I'll fix that.

**Review by indigo423 — APPROVED** [review]
LGTM!

**Pull request merged**